com_dpfields/admin/helpers/dpfields.php

```diff
@@ -187,6 +197,14 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 						));
 					}
 
+					/*
+					 * On after field render
+					 * Event allow plugins to modfify the output of the prepared field
+					 */
+					if ($value)
```

Comment:
Should we not fire the event here under any circumstances, event when the $value is empty?